hub/core/dataset/dataset.py

```diff
@@ -2418,6 +2428,9 @@ def get_view(self, id: str) -> ViewEntry:
                     return ViewEntry(q, qds, True)
         raise KeyError(f"No view with id {id} found in the dataset.")
 
+    def load_view(self, id: str):
+        return self.get_view(id).load()
```

Comment:
docstring for this and delete view? also avoid using id as variable name, shadows built-in id